src/Enums.js

```diff
@@ -32,12 +32,31 @@ export const membershipStatus = [
   { key: 'Admin', value: 2 }
 ]
 
+export const officerRole = {
+  DEVELOPMENT: 'Development',
+  PUBLICRELATION: 'Public Relation',
+  EVENTPLANNING: 'Event Planning',
+  PRINTING: '3D-Printing'
+}
+
+export const chairRole = {
+  PRESIDENT: 'President',
+  VICEPRESIDENT: 'Vice President',
+  TREASURER: 'Treasurer',
+  DEVELOPMENTCHAIR: 'Software Development Chair',
+  EVENTPLANNINGCHAIR: 'Event Planning Chair',
+  STOREMANAGER: 'Store Manager',
+  PRINTINGCHAIR: '3D-Printing/Hacking Station Chair'
+}
+
 export function getKey (enumArray, value) {
```

Comment:
do you need getKey and getValue anymore?

Reply:
Overview is using it, so yes